de.tudarmstadt.ukp.dkpro.argumentation.nio/src/main/java/de/tudarmstadt/ukp/dkpro/argumentation/nio/writer/SpanAnnotationException.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) the Department of Informatics, Technische Universität Darmstadt. All Rights Reserved.
+ *
+ * Unauthorized distribution of this file via any medium is strictly prohibited.
+ */
+package de.tudarmstadt.ukp.dkpro.argumentation.nio.writer;
+
+/**
+ * @author <a href="mailto:shore@ukp.informatik.tu-darmstadt.de">Todd Shore</a>
+ * @since May 2, 2016
+ *
+ */
+public class SpanAnnotationException extends Exception {
```

Comment:
What is the purpose of this class? Can't some existing exceptions (i.e. from UIMA) be used instead of creating a new high-level exception inherited directly from Exception?